ani-cli

```diff
@@ -489,16 +489,20 @@ trackma_title="$(printf '%s' "$anime_id" | tr '-' ' ' | awk '{for(i=1;i<=NF;i++)
 		"syncplay"|"/Applications/Syncplay.app/Contents/MacOS/syncplay"|"/c/Program Files (x86)/Syncplay/Syncplay.exe")
 			set -- "$player_fn" "$video_url" -- --referrer="$refr" --force-media-title="$trackma_title $episode"
 			;;
+        iina)
```

Comment:
imo it would be preferable to check for the existence of iina in /Applications/ as well, since some people might already have iina installed, but not through homebrew; you can check how i used an or statement for syncplay to achieve that

Reply:
yes ofc, homebrew is required for the other dependencies, however some people might have already installed iina previously, so to make it more user friendly it'd be better to have support for regular installations as well